src/freenet/node/NewPacketFormat.java

```diff
@@ -30,7 +31,7 @@
 public class NewPacketFormat implements PacketFormat {
 
 	private final int hmacLength;
-	private static final int HMAC_LENGTH = 10;
+	public static final int HMAC_LENGTH = 10;
```

Comment:
What justifies making this public? This should be of no interest to classes other than `NewPacketFormat`.